src/libostree/ostree-repo-pull.c

```diff
@@ -4060,6 +4101,62 @@ ostree_repo_pull_with_options (OstreeRepo             *self,
       ostree_async_progress_set_status (pull_data->progress, buf->str);
     }
 
+  if (bytes_transferred > 0)
+    {
+#ifdef HAVE_LIBSYSTEMD
```

Comment:
It'd just be an empty if block, no?  But yeah, it's cleaner to have the conditional enclose all of it.  I also folded in the conditional for `remote_name` and de-indented.